### PR TITLE
CHK 1951:  feat post transaction  v2

### PR DESCRIPTION
**Pull request by Fernando-Granato**
#### List of Changes
Added new **POST v2/transaction**  with orderId field into request body. 

#### Motivation and Context
Those modifications has been made in order to introduce api POST transactions versioning

#### How Has This Been Tested?
The tests were performed locally and DEV environment

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/java/it/pagopa/transactions/commands/bean/NewTransactionRequestData.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package it.pagopa.transactions.commands.bean;
```
Comment (giovanniberti) [inline src/main/java/it/pagopa/transactions/commands/bean/NewTransactionRequestData.java:1]: What's the difference between `it.pagopa.transactions.commands.bean` and `it.pagopa.transactions.commands.data`?

File: src/test/java/it/pagopa/transactions/controllers/v2/TransactionsControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+    void shouldGetOk() {
+        TransactionId transactionId = new TransactionId(TransactionTestUtils.TRANSACTION_ID);
+        try (MockedStatic<UUID> uuidMockedStatic = Mockito.mockStatic(UUID.class)) {
+            uuidMockedStatic.when(UUID::randomUUID).thenReturn(transactionId.uuid());
+            String RPTID = "77777777777302016723749670035";
+            String EMAIL = "mario.rossi@email.com";
+            ClientIdDto clientIdDto = ClientIdDto.CHECKOUT;
+            NewTransactionRequestDto newTransactionRequestDto = new NewTransactionRequestDto();
+            newTransactionRequestDto.addPaymentNoticesItem(new PaymentNoticeInfoDto().rptId(RPTID));
+            newTransactionRequestDto.setEmail(EMAIL);
```
Comment (giovanniberti) [inline src/test/java/it/pagopa/transactions/controllers/v2/TransactionsControllerTest.java:85]: Missing `setOrderId`?

File: src/main/java/it/pagopa/transactions/commands/bean/NewTransactionRequestData.java (hunk 1 above)
Comment (Fernando-Granato) [reply src/main/java/it/pagopa/transactions/commands/bean/NewTransactionRequestData.java]: Fixed with 815b383d2b052af9a6a0103e440139ec9e4afdc4.
Effectively the package bean was useless.
Thank you 🙂

File: src/test/java/it/pagopa/transactions/controllers/v2/TransactionsControllerTest.java (hunk 2 above)
Comment (Fernando-Granato) [reply src/test/java/it/pagopa/transactions/controllers/v2/TransactionsControllerTest.java:85]: Fixed with 6c6ee0ed4847203dd8f5a2d0ea6f509b23709f1f

**Review by pietro-tota — APPROVED** [review]

**Pull request merged**